app/src/main/java/com/gaurav/avnc/ui/prefs/PrefsActivity.kt

```diff
@@ -92,7 +92,21 @@ class PrefsActivity : AppCompatActivity(), PreferenceFragmentCompat.OnPreference
         }
     }
 
-    @Keep class Input : PrefFragment(R.xml.pref_input)
+    @Keep class Input : PrefFragment(R.xml.pref_input) {
+        override fun onCreate(savedInstanceState: Bundle?) {
+            super.onCreate(savedInstanceState)
+
+            val canChangePtrIcon = Build.VERSION.SDK_INT >= Build.VERSION_CODES.N
```

Comment:
```suggestion
            val canChangePtrIcon = Build.VERSION.SDK_INT >= 24
```

This is just my personal preference, so if you want, you can ignore this suggestion.